### ISSUE TITLE
Token Parser is documented incorrectly

### ISSUE THREAD

**Issue by gronostajo** [aptoma/twig-markdown#27]
Hello! I'm working on a markdown-related Symfony bundle and I've stumbled upon this library. It looks very promising, although I'm having a problem with the Token Parser.

README recommends using it like this:

```php
use Aptoma\Twig\Extension\MarkdownEngine;
use Aptoma\Twig\TokenParser\MarkdownTokenParser;

$engine = new MarkdownEngine\MichelfMarkdownEngine();

$twig->addTokenParser(new MarkdownTokenParser($engine));
```

However, **`MarkdownTokenParser` doesn't have a constructor that accepts `MarkdownEngineInterface`**, which left me quite puzzled. I looked at `MarkdownTokenParserTest` hoping that it would clear things up, but it looks like **the test doesn't use `MarkdownTokenParser` at all**.

So to sum things up:

- README needs an update
- `MarkdownTokenParser` probably needs a real test
